Fix Those Includes!

ckissane opened the pull request:
!

ckissane committed:
Fix Those Includes!

!

DethRaid reviewed (changes requested):
Please restore the terrain shaders

ckissane committed:
Back to normal shader

Put them back (changed endings though) (without changed endings wont
load with other shaders separate)
